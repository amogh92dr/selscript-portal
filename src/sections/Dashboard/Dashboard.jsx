import React from 'react';

// Sample layout skeletons:
/*
const BothColumns = () => (
  <>
    <div className="dark-blue mx2 mb2">Page Title</div>
    <div className="mx2 mb2 flex">
      <div className="card leftCol dark-blue">
        <div className="sectionTitle pb2">Left Bar</div>
      </div>
      <div className="mx2 content">Page Contents</div>
      <div className="card rightCol dark-blue">
        <div className="sectionTitle pb2">Right Bar</div>
      </div>
    </div>
  </>
);

const LeftColumn = () => (
  <>
    <div className="dark-blue mx2 mb2">Page Title</div>
    <div className="mx2 mb2 flex">
      <div className="card leftCol dark-blue">
        <div className="sectionTitle pb2">Left Bar</div>
      </div>
      <div className="ml2 content">Page Contents</div>
    </div>
  </>
);

const RightColumn = () => (
  <>
    <div className="dark-blue mx2 mb2">Page Title</div>
    <div className="mx2 mb2 flex">
      <div className="mr2 content">Page Contents</div>
      <div className="card rightCol dark-blue">
        <div className="sectionTitle pb2">Right Bar</div>
      </div>
    </div>
  </>
);
*/

// No columns
const Dashboard = () => (
  <div className="card sectionWrapper">
    <div className="dark-blue mx2 mb2">Page Title</div>
    <div className="mx2 mb2 flex">
      <div className="mx2 content">Page Contents</div>
    </div>
  </div>
);

export default Dashboard;
