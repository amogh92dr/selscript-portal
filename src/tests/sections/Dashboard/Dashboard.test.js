import React from 'react';
import renderer from 'react-test-renderer';

import Dashboard from '../../../sections/Dashboard/Dashboard';

describe('Dashboard', () => {
  it('renders the component correctly', () => {
    const component = renderer.create(<Dashboard />);
    expect(component).toMatchSnapshot();
  });
});
