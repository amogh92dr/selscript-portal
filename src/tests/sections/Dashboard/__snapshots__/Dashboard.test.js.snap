// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders the component correctly 1`] = `
<div
  className="card sectionWrapper"
>
  <div
    className="dark-blue mx2 mb2"
  >
    Page Title
  </div>
  <div
    className="mx2 mb2 flex"
  >
    <div
      className="mx2 content"
    >
      Page Contents
    </div>
  </div>
</div>
`;
